gemini/security/colabs/GenAI_&_LLM_Security_(v2_Gemini)_For_Developers_Prompt_Attacks,_Mitigation_Techniques_&_Protection_Strategies.ipynb

```diff
@@ -0,0 +1,2537 @@
qq18XwwOKK95th5HXB6M+gPSpqn6hno4R5y00MoRB7Jppd5zAh6oRtjcLEUHiBnyh+QXcxGpCzrqgfqqMEa9w9JCvt8/lrEYr955p44sEjXAQ4DBMXc9RStooNQE5tQKWY4oKwcaKcvxyjkGnLAkhmjYgz6KQjxK0YzToYQszfmUThRgcEAnjixiGgwOFCGW0lUPHDwIjyc1lI3Y4b1i+XKlh8L96UyBUsGBCFGEfQMj0o3BgnmziTJbqLgyDA7IJzmDA/JpKZ8lBgfk07LylhgcqLwP1DwCBgfU7B1lxsbggDI6a6EXBge04CXlxsjggHJaq6knBgfU5A3tjIXBAe34qpSRMjhQimrytglQtNVBiroqyvJlizE87JQWjxkckFdnpaxFYUXE3ISIqRExo03CP/L1baYABa0ED7TXRWnR3IuB/njqgUgkkq+pqq4HKTXB8FgNRRmox5C/hlIT5I+8IObeQakPti4xYklrEF63k1J25I+CoKqJ82BkUYDBAVlk1JQRBgdU5K6X9ryMXS+8gBd278ael1/BmN+fdXT2qiqsWb0Ks2fNwupVq3DG6afBbicEjgsrUKACpYIDw0TZ+omytdlsOHHbJhiN+fMfFTikgqsxOFCwVHkrMjiQV6IKVGBwoAKil61LBgfKJq0uDDM4oAs3TmsSDA5MSz5dNWZwQFfunPZkGByYtoSaNMDggCbdVvFBMzhQcReUdQAMDpRV3rzGxW7qHtpZLVIU1NfXY/OGNXjp5dcYHMirnLoriFQFESOlLiCAIGaqIXgg/wK6CNHeUCvgAaDaMgancxgjTieCFJVXPMWbbhGRAEK0kG+ijuw2A0W3ACLhsDw7+mmAvrAVvZ46dLsbpIgDhYzXQUtNi9oN2L7UQHP2YYiAAbfbjSinJihEPt3VYXBAdy7NOyEGB/JKVN4KXV3duOtHP8LjTz6NUDBQUmc2exXOfvNZeMeFF2De3Lkl2eBGM0uBUsCBsUAQQyNuSahN69eioYHulipQGByQT3QGB+TTUj5LDA7Ip2XlLTE4UHkfqHkEDA6o2TvKjI3BAWV01kIvDA5owUvKjZHBAeW0VlNPDA6oyRvaGQuDA9rxVSkjZXCgFNXkazM44kIgEILRZMKObZthtVoYHJBP3gpbMiBKqQsi5kYCCBppLOaCAAC7TcADMTTXRDHmG6FoFCPwjY4SXBItaT7EpsATtGFkrJrSBthpo54V7Q2ijwhshjG4fV7aWDpWkm3RKBI1whW0o5uiDPT56gpKTWAkaKHZEcPa+SasnUtRFoIuaZ6jYp5iwFxmpAIMDsw8tzM4UCGfeykPzA9++CP89ne/k3UE55z9Zlz5oStoUZd+y3BhBXIoUCw4ECW6tneQUhTQjdDsjnZKl7Ekh+Xyn2ZwQD6Nm1tbUUXgUTElGArD5R5PoSJuGAUGy0VGBeK3Yg5HNexWa1F2Y92HYb3zWylt7lp4Aw43b085xwfKKZAODoQ3bkPsosumPQDj/3wDpmNHJ+zsrV6Be1d9beKY32hDgWzgQPD0t8Bw+jlTTiD2r4dgfSj1/vFLG+9BkHZLcNGWAungQKRjLqJXf7rwSTz8f7A8+teU+p/d9L+gp5sp5/hA/QqkgwORBYsQveITBQ88Rg8VrTd9MqX+r2ddip2zz0s5xwfaUCAbOBD4j+tgXJSZsjAWjcD6hWtTJvantgvwxLxLUs7xgfoVyAYOBC7/KIwLl2UOnnYhWv7rupTzf2y7EP+e9+6Uc3ygfwXSwYHQtlOAt7+rqImLv0AHh5xFteHKyihgp1VKR01xz2yUGZl6exELm1aLmR5VTe9Z1ShFWx2hqKuirF61HO2tLdJ7jjggyaCb/6IxqwQORC2UusBgLyh1gckYQ1u9Aa20uI6IB8O0E9/l8UgRAooRRnxWvSEb9g21Sov60ZhR+tzW2mJYv9CIE5cZEKZF+2PHezFGkQ2KLcGIGf2+anR7GiUwQUAE+YqF/oyc12LAxsUGLGwKweMaxuCIkza7BvM15es6VyD+tBqUtno+Fszjjcs6d7c0PQYHKuDlp55+Bl/52k3weOK7t+UegqPWgSs/fAXeeu5b5DbN9nSiQLHggJMWin2jYzBbLNhxwmaYibatVGFwQD7lLbZqWlswl26QwYHStcvZMnErlnjNWTHjgqmnC+0/vzPlPIMDKXIofpAODnhWbYTnze+Y9jia7r8D9r5jE3YYHJiQQlNvsoEDI9vPgP+k06ecR9Uz/0LjE6mLxQwOTCmZai+mgwOBlg4Mvfeagsdb/cTDaHjmnyn1GRxIkUMzB+ngwNjs+Rh+14cLHr/BP4pZd6YCZAwOFCyf6ipmAwf6LvogIvOywNsEDsy+9Qspc2BwIEUOzRxkAwf6Lrqc/L4ocw4EDsz+3hdTzjM4kCLHjDlIBwdc60+A74y3FzX/4v/yLMo8V56OAuycktSz261onkaUVGnzlEhRQM+8WpqbsG7NyolxMDgwIYVu3sRgRNTooOgDzYiYHAQP5F9gF1hKbU0MsxyAwx6A2+WkKL0jCFB0APFjW0gRC/mvDLXjqLMpo3q1NYbXrzbj5FUm9Pf342jX8YJ3+8dggDdoRQ+lJjjuqYeP4IRCxlRPjNLSOQZsXWSAzUhAxLCTUxNkeGbmnkj8OmJwYOZ8BhgcUNjXP/3Z/fjhXT9SpNczTj8NX/z85xTpizvRlgLFgAOBYIhCErmkCYqbZXHTXMly8NARHDx4ENFIqJLD0EXfDA6o0Y2JW7HEa+FjZHCgcK2UqsnggFJKa7MfBge06Tc5R83ggJxqatsWgwPa9p/co2dwQG5FtWGPwQFt+Elto2RwQG0ekXk8xT8WkHkA2jRnoYgDbc2lR+LtpwgcIYq2aSY7IkWB2Ty54YbBAW1+JvKPmlIXwE7RB5oQoegDMFgLWqi3Ww1or4+hqTqMYMCNwcFh+CjKdLSA1AXhqAn/PLIEYxFLxvBMxC4sbDPg/adZ6LMYwu49r9Jz8EhG
```

Comment:
It's usually preferred to use official libraries instead of raw REST API Calls. Can you try using their library? https://pypi.org/project/virustotal-python/